docs/how-to-guides/new-user-guides/backup-restore-and-disaster-recovery/restore-rancher.md

```diff
@@ -8,39 +8,24 @@ title: Restoring Rancher
 
 This page outlines how to perform a restore with Rancher.
 
-:::note Important:
+Follow the instructions from this page when:
+- the running Rancher is at the same version as when the backup was made
+- the local cluster is the same as where the backup was made 
 
-* Follow the instructions from this page for restoring rancher on the same cluster where it was backed up from. In order to migrate rancher to a new cluster, follow the steps to [migrate rancher.](migrate-rancher-to-new-cluster.md)
-* While restoring rancher on the same setup, the operator will scale down the rancher deployment when restore starts, and it will scale back up the deployment once restore completes. So Rancher will be unavailable during the restore.
-* If you need to restore Rancher to a previous version after an upgrade, see the [rollback documentation.](../../../getting-started/installation-and-upgrade/install-upgrade-on-a-kubernetes-cluster/rollbacks.md)
-
-:::
-
-## Additional Steps for Rollbacks with Rancher v2.6.4+
-
-Rancher v2.6.4 upgrades the cluster-api module from v0.4.4 to v1.0.2. Version v1.0.2 of the cluster-api, in turn, upgrades the Cluster API's  Custom Resource Definitions (CRDs) from `cluster.x-k8s.io/v1alpha4` to `cluster.x-k8s.io/v1beta1`. The CRDs upgrade to v1beta1 causes rollbacks to fail when you attempt to move from Rancher v2.6.4 to any previous version of Rancher v2.6.x. This is because CRDs that use the older apiVersion (v1alpha4) are incompatible with v1beta1.
+:::note Important
 
-To avoid rollback failure, the following Rancher scripts should be run **before** you attempt a restore operation or rollback: 
+While restoring rancher on the same setup, the operator will scale down the rancher deployment when restore starts, and it will scale back up the deployment once restore completes. So Rancher will be unavailable during the restore.
```

Comment:
```suggestion
While restoring Rancher on the same setup, the operator will scale down the Rancher deployment when restore starts, and it will scale back up the deployment once restore completes. Rancher will be unavailable during the restore.
```